src/js/components/ToggleGroup/StyledToggleGroup.js

```diff
@@ -48,4 +48,5 @@ export const StyledButton = styled(Button)`
       `border: none;
     box-shadow: none;`}
   }
+  ${(props) => props.theme.toggleGroup?.container?.extend};
```

Comment:
Should this go on the container around the buttons instead of the buttons themselves?

Reply:
yes I misunderstood Taylor thought it was just the one button.

Reply:
Ill adjust